client_2x/src/App.vue

```diff
@@ -16,21 +16,27 @@
   </v-app>
 </template>
 
-<script>
-import Vue from "vue";
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
 import Toolbar from "@/components/Toolbar/Toolbar.vue";
 
-export default Vue.extend({
-  name: "App",
-  components: { Toolbar },
-  data: () => ({})
-});
+@Component({
+  name: "OctoFarm",
+  components: {
+    Toolbar
+  }
+})
+
+export default class OctoFarm extends Vue {
+
+};
 </script>
+
 <style>
 main {
   background-color: black;
   background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
-    url("~@/assets/images/bg.jpg");
```

Comment:
We have the paths entry in `tsconfig.json` for shorthand paths. Can look at it later.